Abdul/BinaryTree.py

```diff
@@ -0,0 +1,59 @@
+class TreeNode: 
+    def __init__(self, name, distance) -> None:
+        self.name = name 
+        self.distance = distance
+        self.left = self.right = None 
+
+cities = TreeNode("Seattle", 0)
+cities.left = TreeNode("Denver", 1305)
+cities.right = TreeNode("New York", 2401)
+cities.right.right = TreeNode("Boston", 216)
+cities.left.left = TreeNode("Miami", 2064)
+cities.left.right = TreeNode("Los Angles", 1016)
+cities.left.right.left = TreeNode("Dallas", 1435)
+cities.left.right.left.right = TreeNode("Minneapolis", 990)
+
+def find_path(city: TreeNode, destination: str, totalPath: str) -> str: 
+	# return if city is None
+	if not city:
+		return 
```

Comment:
Here you are not returning anything and your function in line 31 is returning s1 or s2. This is not consistent.